web-console/knikubevirt/Create-vm/step-6-results/wizard-results.md

```diff
@@ -0,0 +1,49 @@
+# Create Virtual Machine Wizard - Results Step
+Once the user has clicked "Create virtual machine" they will see the progress state of the VM being created.
+
+![user clicks create button](img/create-vm-inline-wizard-in-progress.png)
+
+Alternatively, The user could navigate away from this page, In that case they'd see the VM in the list in a starting state. This will rely on the VM successfully being created without issue. For more info on error/success states see below.
+
+![vm is being started](img/vm-starting-state.png)
+
+## Success State
+If the process is successful they will see a success state on the Results page.
+
+There is no navigation provided on this page because the user can not go back and make changes at this point. 
+They are provided actions to either go to the VM details or view the VM list.
+
+![VM success](img/create-vm-inline-wizard-success.png)
+
+If they are looking at the list view they will see a status of `running` or `off` if the creation was successful. 
+
+## Error State
+
+If there is an error during the creation of the VM the results page will show a breakdown of the errors with links to the steps where the error exists. 
+
+They will also see the YAML(s) available for review.
+
+![VM Error](img/create-vm-inline-wizard-error.png)
+
+In the step that contains the error they would see the error state of the input they need to address.
+
+![vm error state](img/create-vm-inline-wizard-input.png)
```

Comment:
This is small comment and hopefully a quick fix. I've noticed that across the flow we are using both old and new PatternFly blues. If you have the chance, it would be great to update all of the blues with the new PF blue. https://www.patternfly.org/v4/design-guidelines/styles/colors